python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -254,6 +261,7 @@ def _get_sg_software_entities(self):
             "image",
             "sg_engine",
             "sg_versions",
+            "sg_products",
```

Comment:
Not your fault, but it's going to be annoying to fix all of these when we have the official fields. :(